next-app/src/components/views/Main/Main.tsx

```diff
@@ -0,0 +1,90 @@
+import * as S from 'components/views/Main/Main.style'
+
+const Home = () => {
+  return (
+    <S.Wrapper>
+      <S.Container>
+        <S.ContentsWrapper>
+          <S.Contents>
+            <S.ImageWrapper />
+            <S.BoardWrapper>
+              <div>
+                <S.Title>온보딩 컨텐츠 제목</S.Title>
+                <S.Text>
+                  <span>
+                    At vero eos et accusamus et iusto odio dignissimos ducimus
+                    qui blanditiis praesentium voluptatum deleniti atque
+                    corrupti quos dolores et quas molestias.
+                  </span>
+                  <br />
+                  <br />
+                  <span>
+                    At vero eos et accusamus et iusto odio dignissimos ducimus
+                    qui blanditiis praesentium voluptatum deleniti atque
+                    corrupti quos dolores et quas molestias.
+                  </span>
```

Comment:
```suggestion
                  </p>
```
여기서 p태그를 클로징 해주시면 될 것 같습니당!